README.md

```diff
@@ -1,332 +1,72 @@
-2) Custom models require a simple webserver in order to make predictions. We recommend putting this in
-your environment so that you can reuse it with multiple models. The webserver must be listening on port 8080 and implement the following routes:
-   > **Note: `URL_PREFIX` is an environment variable that will be available at runtime. It has to be read and pasted into the routes.**  
-   > **Note: Refer to the complete API specification: [drum_server_api.yaml](custom_model_runner/drum_server_api.yaml), you can also open it rendered in the [Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/datarobot/datarobot-user-models/master/custom_model_runner/drum_server_api.yaml).**
-    1) Mandatory endpoints:
-        1) `GET /URL_PREFIX/` This route is used to check if your model's server is running.
-        2) `POST /URL_PREFIX/predict/` This route is used to make predictions.
-    2) Nice-to-have extensions endpoints:  
-        1) `GET /URL_PREFIX/stats/` This route is used to fetch memory usage data for DataRobot Custom Model Testing.
-        2) `GET /URL_PREFIX/health/` This route is used to check if model is loaded and functioning properly. If model loading fails error with 513 response code should be returned.  
-           Failing to handle this case may cause backend k8s container to enter crash/restart loop for several minutes.
-3) An executable `start_server.sh` file is required to start the model server.
-4) Any code and `start_server.sh` should be copied to `/opt/code/` by your Dockerfile
+* [Custom models walkthrough](https://community.datarobot.com/t5/knowledge-base/working-with-custom-models/ta-p/6082)
+* Code examples:
+    * [Custom inference models templates](https://github.com/datarobot/datarobot-user-models/tree/master/model_templates)
+    * [Environment Templates](https://github.com/datarobot/datarobot-user-models/tree/master/public_dropin_environments)
+* How to define a custom inference model:
+    * [Assembling a code folder](https://docs.google.com/document/d/154C_dDDZJlBClK4DjCOXQJlos6jszicgOmcCX14Mm_s/edit#heading=h.q52wbalbzdy9)
+    * [Define a structured model](https://docs.google.com/document/d/154C_dDDZJlBClK4DjCOXQJlos6jszicgOmcCX14Mm_s/edit#heading=h.rd7ksjij0hhn)
+    * [Define an unstructured model](https://docs.google.com/document/d/154C_dDDZJlBClK4DjCOXQJlos6jszicgOmcCX14Mm_s/edit#heading=h.wg9ykdry860i)
+    * [Test a model with DRUM locally](https://docs.google.com/document/d/154C_dDDZJlBClK4DjCOXQJlos6jszicgOmcCX14Mm_s/edit#heading=h.wx9siz7rht7w)
+    * [Build your own environment](https://docs.google.com/document/d/154C_dDDZJlBClK4DjCOXQJlos6jszicgOmcCX14Mm_s/edit#heading=h.d59dwsyskndu)
+    * [Upload, manage, and deploy a model in DataRobot](https://app.datarobot.com/docs/mlops/deployment/custom-models/custom-inf-model.html)
 
+Other sources:
+* There is a chance that the task you are looking for has already been implemented. 
```

Comment:
Let's replace "task" with "model" in here

Reply:
done